services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -36,6 +35,11 @@ func handlePost(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int
 	// When ContentLength is 0 that means request body is empty and
 	// storage quota is unlimited
 	var err error
+	client, err := newClient()
+	if err != nil {
+		klog.Errorf("failed to create new client. %v", err)
+	}
```

Comment:
you shouldn't be creating kubeclient for every request, create it in main and use that.

Reply:
@Mrudraia I believe main already creates a client and some other endpoints accept that client. Can you please follow that pattern

Reply:
Acknowledge